include/oc_uuid.h

```diff
@@ -141,6 +142,18 @@ void oc_gen_uuid(oc_uuid_t *uuid) OC_NONNULL();
 OC_API
 bool oc_uuid_is_equal(oc_uuid_t first, oc_uuid_t second);
 
+#ifdef OC_HAS_FEATURE_BRIDGE
+/**
+ * @brief Check if the uuid is null or not
+ *
+ * @param uuid A uuid to be checked
+ * @return true If the uuid is null
+ * @return false Otherwise
+ */
+OC_API
+bool oc_uuid_is_nil(const oc_uuid_t *uuid);
```

Comment:
lets use value sematics, so we're consistent with `oc_uuid_is_equal`

```C
bool oc_uuid_is_nil(const oc_uuid_t *uuid);
```

->


```C
bool oc_uuid_is_nil(oc_uuid_t uuid);
```

Reply:
done